src/components/Dashboard/Github/dashboard.tsx

```diff
@@ -345,39 +379,72 @@ const DashboardGithub: React.FunctionComponent = () => {
                           popperProps={{ position: 'end' }}
                         >
                           <DropdownList>
-                            <DropdownItem key="view-pr" to={pr.html_url} target="_blank" rel="noopener noreferrer">
-                              View PR
-                            </DropdownItem>
-                            {pr.state === 'open' && (
-                              <DropdownItem key="edit-contribution" onClick={() => handleEditClick(pr)}>
-                                Edit contribution
-                              </DropdownItem>
+                            {!draftContributions.find((draft) => draft.branchName == pr.head.ref) && (
+                              <>
+                                <DropdownItem key="view-pr" to={pr.html_url} target="_blank" rel="noopener noreferrer">
+                                  View PR
+                                </DropdownItem>
+                                {pr.state === 'open' && (
+                                  <DropdownItem key="edit-contribution" onClick={() => handleEditClick(pr)}>
+                                    Edit contribution
+                                  </DropdownItem>
+                                )}
+                                {pr.state === 'closed' && (
+                                  <DropdownItem key="edit-contribution" isDisabled>
+                                    Edit contribution
+                                  </DropdownItem>
+                                )}
```

Comment:
This could be a single item with `isDisabled={pr.state === 'closed`. Unless I am missing other states where we wouldn't want to show the button?